### PR TITLE
Switch to helusers admin

### PR DESCRIPTION
**Pull request by Rikuoja**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+git+https://github.com/City-of-Helsinki/django-helusers.git#egg=django-helusers
```
Comment (akx) [inline requirements.txt]: This will always get the `master` version of helusers, with no pinning whatsoever. Is that a good idea?

**Comment by akx** [comment]
This seems to do a whole lot of changes, and the commit message does not explain why, nor does it have refs to an issue or anything. :(

**Comment by akx** [comment]
Tossing ball back to @Rikuoja.

File: requirements.txt [hunk 2]
```diff
@@ -19,4 +19,5 @@ django-nested-admin==3.0.2
 django-geojson==2.9.0
 django-leaflet==0.18.0
 django-ckeditor==5.0.3
-django-autoslug==1.9.3
\ No newline at end of file
+django-autoslug==1.9.3
+django-allauth
```
Comment (JsseL) [inline requirements.txt]: Might be good idea to pin the `django-allauth` too. Pinned versions makes things more robust.

**Pull request closed without merging**